examples/generative/conditional_gan.py

```diff
@@ -0,0 +1,323 @@
+"""
+Title: Conditional GANs
+Author: [Sayak Paul](https://twitter.com/RisingSayak)
+Date created: 2021/07/13
+Last modified: 2021/07/15
+Description: Training a GAN conditioned on class labels to generate handwritten digits.
+"""
+"""
+Generative Adversarial Networks (GANs) let us generate novel samples from noise, adapt
+one domain into another, and so on. For the first case, the noise is generally sampled
+from a normal distribution and then it goes through a series of transformations that turn
+the noise into something plausible (image, video, audio, etc.). Now, if we wanted
+to generate novel hand-written digits using a GAN. But we also wanted to *condition* its
+outputs on the class of digit we want to generate i.e. zero, one, two, three, and so on.
+[Vanilla DCGAN](https://arxiv.org/abs/1511.06434) won't allow us to achieve this because
+it does have that conditional aspect in its setup.
```

Comment:
```suggestion
the noise into something plausible (image, video, audio, etc.). Now, if we want
to generate novel hand-written digits using a GAN, and also want to *condition* its
outputs on the class of digit we want to generate, i.e. zero, one, two, three, and so on, then
[Vanilla DCGAN](https://arxiv.org/abs/1511.06434) won't allow us to achieve this because
it does have that conditional aspect in its setup.
```